enterprise/management/commands/email_drip_for_missing_dsc_records.py

```diff
@@ -170,8 +175,12 @@ def handle(self, *args, **options):
                 course_id=course_id,
                 enterprise_customer=enterprise_customer
             )
+            course_accessed = False
+            if is_course_accessed and is_course_accessed(ec_user.user, course_id):
```

Comment:
Wouldn't this flag `is_course_accessed` always be False since we are explicitly setting it False in the above statement?